pkgs/development/python-modules/pc-ble-driver-py/default.nix

```
@@ -1,45 +1,24 @@
-{ stdenv, buildPythonPackage, fetchpatch, fetchFromGitHub,
-  python, cmake, git, swig, boost, udev,
-  setuptools, enum34, wrapt, future }:
+{ stdenv, fetchFromGitHub, cmake, git, swig, boost, udev, pc-ble-driver, python37Packages }:
 
-buildPythonPackage rec {
+with python37Packages; buildPythonPackage rec {
```

```suggestion
buildPythonPackage rec {
```
Python packages can be built for different python versions.  For example `nix-build -A python38Packages.pc-ble-driver-py`.  This file should not import `python37Packages`.

---

Fixed. Same for nrfutil? It doesn't build on all Python 3 versions, and not at all on Python 2.

---

For applications it's fine to hardcode a python version.  It's only the libraries where you might want to use them with a different python version.

Please squash the "Don't depend on a fixed Python version" commit into the "nrfutil: ..." commit.

---

Ok, done! (I'm assuming you meant the pc-ble-driver-py commit)

---

Oh yes, of course.